doc/Architecture/arc-tec_x-road_technologies.md

```diff
@@ -168,28 +176,28 @@ See [[ARC-CP]](#ARC-CP) for the configuration proxy details.
+| **Technology**           | **Signer** | **Proxy** | **Password Store** | **Message Log** | **Metadata Services** | **Database** | **Configuration Client** | **User Interface frontend** | **REST API** | **Monitor** | **Environmental Monitoring Service** | **Operational Monitoring Buffer** | **Operational Monitoring Services** |
+|--------------------------|:----------:|:---------:|:------------------:|:---------------:|:---------------------:|:------------:|:------------------------:|:---------------------------:|:------------:|:-----------:|:------------------------------------:|:---------------------------------:|:-----------------------------------:|
+| Java 11                  |     X      |     X     |                    |        X        |           X           |              |            X             |                             |      X       |      X      |                  X                   |                 X                 |                  X                  |
+| C                        |            |           |         X          |                 |                       |              |                          |                             |              |             |                                      |                                   |                                     |
+| Logback                  |     X      |     X     |                    |        X        |           X           |              |            X             |                             |      X       |             |                  X                   |                 X                 |                  X                  |
+| Akka 2                   |     X      |     X     |                    |        X        |                       |              |                          |                             |      X       |      X      |                  X                   |                 X                 |                                     |
+| Embedded Jetty 9         |            |     X     |                    |                 |                       |              |                          |                             |              |             |                                      |                                   |                                     |
+| Javascript               |            |           |                    |                 |                       |              |                          |              X              |              |             |                                      |                                   |                                     |
+| PostgreSQL 10            |            |           |                    |                 |                       |      X       |                          |                             |      X       |             |                                      |                                   |                                     |
```

Comment:
Should PostgreSQL 12 (Ubuntu 20.04) and PostgreSQL 14 (Ubuntu 22.04) be added? Also, PostgreSQL versions used by RHEL7 and RHEL8 should be checked.